solutions/scripts/delayed.js

```diff
@@ -2,27 +2,26 @@
 import {
   loadScript,
   sampleRUM,
-  fetchPlaceholders,
+  // fetchPlaceholders,
   getMetadata,
 } from './lib-franklin.js';
 
 // eslint-disable-next-line import/no-cycle
 import {
-  getLanguageCountryFromPath,
+  // getLanguageCountryFromPath,
   pushProductsToDataLayer,
   pushToDataLayer,
-  getEnvironment,
+  // getEnvironment,
   openUrlForOs,
 } from './scripts.js';
 import { loadBreadcrumbs } from './breadcrumbs.js';
 
 // Core Web Vitals RUM collection
 sampleRUM('cwv');
 
-const LANGUAGE_COUNTRY = getLanguageCountryFromPath(window.location.pathname);
+/* const LANGUAGE_COUNTRY = getLanguageCountryFromPath(window.location.pathname);
```

Comment:
Let's uncomment this and reference the new tag by adding an entry to the placeholders file.

Reply:
@vtsaplin  you mean something like this?

const LANGUAGE_COUNTRY = getLanguageCountryFromPath(window.location.pathname);
// const LAUNCH_URL = 'https://assets.adobedtm.com';
const ENVIRONMENT = getEnvironment(window.location.hostname, LANGUAGE_COUNTRY.country);
// Load Adobe Experience platform data collection (Launch) script
// const { launchProdScript, launchStageScript, launchDevScript } = await fetchPlaceholders();
switch (ENVIRONMENT) {
  case 'prod':
    loadScript('https://assets.adobedtm.com/8a93f8486ba4/e7dc9e6549e5/launch-42a4d88f6f31.min.js'); break;
  case 'stage':
    loadScript('https://assets.adobedtm.com/8a93f8486ba4/e7dc9e6549e5/launch-d0a308d8ad78-staging.min.js'); break;
  default:
    loadScript('https://assets.adobedtm.com/8a93f8486ba4/e7dc9e6549e5/launch-aef7ddf31563-development.min.js'); break;
}

Reply:
@cuzuco2 Let's keep the code as it was before but make a tmp change in the `scripts.js`:
```
export function getEnvironment(hostname, country) {
  if (hostname.startsWith('pull_433')) {
    return 'dev';
  }
  if (hostname.includes('hlx.page') || hostname.includes('hlx.live')) {
    return 'stage';
  }
  if (hostname.includes(`.${country}`)) {
    return 'prod';
  }
  return 'dev';
}
```
You'd also need to change the dev ID in placeholders.